micro-demos/services/demo.sh

```diff
@@ -15,18 +15,18 @@
 
 . $(dirname ${BASH_SOURCE})/../util.sh
 
-if kubectl --namespace=demos get rc hostnames >/dev/null 2>&1; then
-    desc "Revisit our replication controller"
-    run "kubectl --namespace=demos get rc hostnames"
+if kubectl --namespace=demos get rs hostnames >/dev/null 2>&1; then
+    desc "Revisit our replica set"
+    run "kubectl --namespace=demos get rs hostnames"
 else
-    desc "Run some pods under a replication controller"
-    run "kubectl --namespace=demos run hostnames \\
```

Comment:
I'll copy my version to another folder, perhaps "services-yaml/".

I'll convert the RC to RS. 

I'm not sure about deployments, because usually we introduce services before deployments. If people want to show `expose` works on deployments, they can write their own demo.